data_steward/analytics/table_metrics/weekly_scripts/erroneous_dates.py

```diff
@@ -0,0 +1,583 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# ### This notebook is intended to show the percentage of rows where there are 'erroneous dates' in the 6 canonical tables. The 6 canonical tables are as follows:
+# - Condition Occurrence
+# - Procedure Occurrence
+# - Visit Occurrence
+# - Drug Exposure
+# - Measurement
+# - Observation
+#
+# ### Erroneous dates are those that precede 1900 for the observation table or precede 1980 for all other tables
+
+# +
+from google.cloud import bigquery
+
+# %reload_ext google.cloud.bigquery
+
+client = bigquery.Client()
+
+# %load_ext google.cloud.bigquery
+
+# +
+from notebooks import parameters
+DATASET = parameters.LATEST_DATASET
+
+print("Dataset to use: {DATASET}".format(DATASET = DATASET))
+
+# +
+#######################################
+print('Setting everything up...')
+#######################################
+
+import warnings
+
+warnings.filterwarnings('ignore')
+import pandas as pd
+import matplotlib.pyplot as plt
+# %matplotlib inline
+import os
+
+
+plt.style.use('ggplot')
+pd.options.display.max_rows = 999
+pd.options.display.max_columns = 999
+pd.options.display.max_colwidth = 999
+
+
+def cstr(s, color='black'):
+    return "<text style=color:{}>{}</text>".format(color, s)
+
+
+print('done.')
+# -
+
+cwd = os.getcwd()
+cwd = str(cwd)
+print("Current working directory is: {cwd}".format(cwd=cwd))
```

Comment:
with f-strings this would be
```suggestion
print(f"Current working directory is: {cwd}")
```

Reply:
I see - I take it that it implicitly places in the variable that is in the {}?